src/components/UserInfo/UserInfo.tsx

```diff
@@ -1,5 +1,20 @@
 import React from 'react';
 
-export const UserInfo: React.FC = () => (
-  <>Put the user here</>
-);
+import { User } from '../../types/user';
+
+type Props = {
+  person: User;
+};
+
+export const UserInfo: React.FC<Props> = ({ person }) => {
+  const {
+    name,
+    email,
+  } = person;
```

Comment:
and here. change name of prop from 'person' on 'user' and good luck. it seems like feature or bugs in tests. i agree with this naming in fact it can works, but....  we can ask today(18/11) on live lesson about. i also want to know why test crashes